client/state/data-layer/wpcom/sites/comments/index.js

```diff
@@ -138,31 +139,43 @@ export const receiveCommentError = ( { dispatch, getState }, { siteId, commentId
 
 // @see https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/comments/
 export const fetchCommentsList = ( { dispatch }, action ) => {
-	const { query } = action;
-
-	if ( ! query || 'site' !== query.listType ) {
+	if ( ! action.query || 'site' !== action.query.listType ) {
```

Comment:
Although, I'd really like to know what would be `listType` use case, as AFAIK it's not used by anyone anywhere...